src/extension.ts

```diff
@@ -102,9 +104,6 @@ export function activate(context: vscode.ExtensionContext) {
 	});
 
 	context.subscriptions.push(listDevicesCommand);
-
-	const schemaPath = vscode.Uri.file(path.join(context.extensionPath, "schema", "schema.v0.json")).toString();
-	updateYamlSchemaAssociations(schemaPath);
```

Comment:
This call no longer exists within this method. Intentional?

If so, when is schemaPath now used?

Reply:
Line 40 - 46. Couldn't find how to get a permalink to it.

When the filePatterns are updated, we `updateYamlSchemaAssociations` and also `updateFileWatcherAndTreeView `